sdk/communication/azure-communication-phonenumbers/swagger/README.md

```diff
@@ -23,8 +23,8 @@ autorest README.md --java --v4 --use=@autorest/java@4.0.2
 
 ### Code generation settings
 ``` yaml
-tag: package-phonenumber-2021-03-07
```

Comment:
I think you need to keep both versions in this file.

Reply:
I don't think both could live together, since it's the param configuration for the Autorest. 
It might cause some conflict (I haven't tested this, I'm just assuming).

Reply:
Oh this is code generation settings, and not for READMEs.

Then yes, I think this is a replacement, not an add.

In that case, we can leave it as-is.